Assets/_scenes/Minigame.unity

```diff
@@ -483,7 +483,7 @@ Rigidbody:
   m_AngularDrag: 0.05
   m_UseGravity: 1
   m_IsKinematic: 0
-  m_Interpolate: 0
+  m_Interpolate: 1
```

Comment:
What does interpolation do? I assume it smooths the ball out, but why did you turn it on? Is there jerkiness without it?